examples/mks.rs

```diff
@@ -3,18 +3,14 @@
 #[macro_use]
 extern crate uom;
 
+use uom::fmt::DisplayStyle::*;
```

Comment:
Personally, I am not overly fond of glob imports and would suggest avoiding them in examples for pedagogical reasons.